src/Tizen.NUI/src/public/Animation.cs

```diff
@@ -436,6 +442,8 @@ public float ProgressNotification
         /// <summary>
         /// Gets or sets the properties of the animation.
         /// </summary>
+        //ToDo : will raise deprecated-ACR, [Obsolete("Deprecated in API9, Will be removed in API11, Please use PropertyList instead")]
+        [SuppressMessage("Microsoft.Naming", "CA1819:This API will be deprecated, so suppressing the warning for now")]
```

Comment:
Hi @AdunFang ,
The second parameter is `RuleId`, so you may need to add `Justification` to write reason.
https://docs.microsoft.com/en-us/visualstudio/code-quality/in-source-suppression-overview?view=vs-2019#suppressmessage-attribute
```
[SuppressMessage("Rule Category", "Rule Id", Justification = "Justification", MessageId = "MessageId", Scope = "Scope", Target = "Target")]
```

And the rule `CA1819` is not in "Naming" category. It is "Performance".
```
[SuppressMessage("Microsoft.Performance, "CA1819:PropertiesShouldNotReturnArrays", Justification = "This API will be deprecated, so suppressing the warning for now")]
```